templates/docs/examples/patterns/forms/_script.js

```diff
@@ -0,0 +1,25 @@
+(function () {
+  var passwordToggleButtons = Array.prototype.slice.call(document.querySelectorAll('.p-form-password-toggle__button'));
+
+  passwordToggleButtons.forEach(function (passwordToggleButton) {
+    var passwordToggleLabel = passwordToggleButton.querySelector('.p-form-password-toggle__label');
```

Comment:
It's a little thing, but I'd be tempted to make this target the element rather than the class, we generally try to keep classes out of JS examples as much as possible, and it'd be more consistent with how you're getting the icon element:
```suggestion
    var passwordToggleLabel = passwordToggleButton.querySelector('span');
```